### DISCUSSION TITLE
Contacts moved into recycle bin

### DISCUSSION THREAD

**Discussion by vovpff** [YetiForceCompany/YetiForceCRM#15598] [How to]
Around month ago my colleague said what he can't find contact via list view or global search what he added in the past. I checked whole  system and found what this contact is in database in recycle bin, I restored contact to active state.

Today I checked contacts recycle bin and found this contact one more time.

In history tab I can see what month ago 26.11 contact was restored by me. I tried to restore it one more time and moved to bin after. All my today actions saved into history. Pls check screenshot:

![image](https://user-images.githubusercontent.com/8545791/147665984-c90c599b-213a-4b65-a2a2-2d4f3bf4c484.png)

How to understand what is going on? I'm confused.

Are there an mechanism in system what automatically change record state of contacts?

I'm on 6.1 now and all my users don't have permissions to archive or move to trash. Only admin and my user (vov) can do this. This happens only with contacts records. Pls help!

**Comment by mariuszkrzaczkowski** [comment]
I have no idea what it could be, it would require system analysis, e.g. handlers, crons and all automation

> Are there an mechanism in system what automatically change record state of contacts?

No

**Reply by vovpff** [reply]
I has only two self written handlers what applicable for contacts. This handler doesn't affect with `u_yf_crmentity.deleted` table column.

%(

**Comment by garryatbrighton** [comment]
If I remember well I had smiliar problem. It was not on production so didint investigate assuming I had deleted records by mistake.
But this sits in my mind whispering It was impossible I have been deleted records. Now doubt doubled

**Reply by vovpff** [reply]
+1
